app_layout_1.R

```diff
@@ -45,13 +48,28 @@ list(
 htmlLabel("Select what you want to view:"),
 htmlBr(),
 htmlBr(),
-# add radio button
+dccRadioItems(
+  id="radio-selection",
+  options=list(
+    list(label = "Categories", value = "category"),
+    list(label = "Mechanics", value = "mechanic"),
+    list(label = "Publishers", value = "publisher")
+  ),
+  value="mechanic"
+),
 htmlBr(),
 htmlLabel("Select elements to view:"),
 htmlBr(),
-htmlBr())))}
-   # add radio button dependent drop down,
-  
+htmlBr(),
+dccDropdown(
+  id = "radio-dependent",
+  options=list(),
+  multi=TRUE,
+  value=NULL,
+  # labelStyle=list("display"="block")
```

Comment:
this exists in the python repo

but if i include it here, it gives me a "param not used"

Reply:
`# labelStyle=list("display"="block")` you mean that line?

Reply:
yeah